src/styles/search-results.css

```diff
@@ -0,0 +1,13 @@
+.search-results {
+  width: 80%;
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  row-gap: 10px;
+  column-gap: 10px;
+}
+
+.search-results_image {
+  height: 300px;
+  width: 300px;
+}
```

Comment:
This styling is setting all images to be squares, which is causing some images to be stretched or squished. I would recommend setting the `width` and `height` properties to `100%` and then setting the `max-height` and `min-height` to `300px` this will fix that problem. You can then add the line `align-self: center;` to have all the image centred nicely, e.g.
```
.search-results_image {
  align-self: center;
  width: 100%;
  max-width: 300px;
  height: 100%;
  max-height: 300px;
}
```